ports/qemu-riscv/mpconfigport.h

```diff
@@ -0,0 +1,50 @@
+#include <stdint.h>
```

Comment:
Can you please add license and copyright headers to this file and other .c/.h files that are missing them?  If you copied the files from `qemu-arm` then please see their updated files which contain license/copyright as of 9bfb4cec6e832c38eb04ad680a2b5e377830d3af

Reply:
The source files copyright and licence comments have been updated as requested.